src/main/java/com/khutircraftubackend/seller/SellerEntity.java

```diff
@@ -4,31 +4,43 @@
 import jakarta.persistence.*;
 import lombok.*;
 
+import java.time.LocalDateTime;
+
 /**
  * Клас Seller є моделлю продавця і відображає таблицю продавців у базі даних.
  */
 
 @Entity
+@Table (name = "seller")
```

Comment:
for a table in db we might want have it in plural: `sellers`, similar to `users` in `UserEntity`